internal/backend_gcs.go

```diff
@@ -0,0 +1,276 @@
+package internal
+
+import (
+	"cloud.google.com/go/storage"
```

Comment:
I forgot. Did we decide to only use the "polished" google api?

Can the polished and the raw api be used side-by-side?

Reply:
Polished API (aka client API). I think we can use the raw and polished api at the same time because the polished API is implemented on top of the raw API. I'd recommend to limit the usage of raw API because they can change at anytime and it can be a breaking change (not backward compatible)